geocoder_tester/base.py

```diff
@@ -112,9 +172,9 @@ def flat_result(self, result):
             out['distance'] = int(dist.meters)
         return out
 
-
-def search(**params):
-    r = http.get(CONFIG['API_URL'], params=params)
+def search(url, **params):
+    r = http.get(url, params=params,
+                 headers={'user-agent': 'geocode-tester/0.0.1'})
```

Comment:
If we put the version here, we should read it from the package itself, but I'll do that later (there is no formal package atm).

Reply:
I've removed the version for now. We can  re-add it, once there is an actual version.